news/2022-02-07-beatmap-spotlights-season-5-winter-2022.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: post
+title: "Beatmap Spotlights Season 5: Winter 2022 - unorthodox rhythm"
+date: 2022-02-07 01:45:00 +0000
+---
+
+Spotlights rivals assemble! After a long break, we are finally back, many things changed and so we hope you all are ready for **Winter Season 2022**! <!-- possibly some unorthodox rhythm related thing -->
+
+![](/wiki/shared/news/banners/beatmap-spotlights.jpg)
+
+With the new season launch, there will be some major changes regarding gameplay aspects, as well as communication related matters.
+
+If you just can't wait to try out the new maps, **the Beatmap Spotlights lobbies are now open for play on the osu!lazer client**. If you've got no idea how to get started with things, read on below!
+
+## The changes
+
+### New spotlights playlists format
+
+We've heard a lot of voices that the old playlists format might've been a little too boring for more experienced players as one or two extras and/or insanes per list is just simply not enough to be interesting. To increase the competition, and at the same time, to keep it also approachable for newer players, we decided to reshuffle how the playlists system work. **From now on, each playlist will last 2 weeks and will include 2 hard, 3 insane, and 4 extra difficulties.** Each season will consist of 3 playlists, after the first run, the playlists will be re-played for a week.
+
+### Themed playlists
+
+But that's not all regarding the playlists changes. **Starting this season, the first playlist of every season will be themed with some kind of gimmick.** We've came to a conclusion that spotlights playlists might've felt a little disconnected, to make them more interesting, we decided to make the first playlist of every incoming season special. **This time, the theme is unorthodox rhythm of any sorts**.
+
+### More transparency
+
+There's been a lot of voices that nobody really knows who's the member and what's going on in the Spotlighs Team. We'd like to face this issue, and make it so everything is a little bit more clear to the outsiders. Each season announcement will now include *[Inside the spotlights team](inside-the-team)* section, listing the most recent team changes. We will also credit respective curators, resposnible for suggesting the maps, in the [picks lists](playlists). There's more to come in that regard, so stay tuned for the next seasons!
+
+## Medals and beatmap packs
+
+Many people stated that they'd like to enjoy some kind of spotlights experience on stable, like it used to be in the past. To make it somehow possible, we will now release beatmap packs along with medal set for each gamemode once all of the playlists are released (in 4 weeks). Have fun jumping on grindset to catch them all!

```

Comment:
Considering there were some plans for releasing retrospective medals beatmap packs for the earlier seasons that were ignored (packs for winter20 spring21 and medals for the prior seasons + summer20) it would be nice to have a mention regarding them here